app/webpacker/packs/application.js

```diff
@@ -13,6 +13,7 @@
 // or the `imagePath` JavaScript helper below.
 //
 require.context('images/flags');
+require.context('images/');
```

Comment:
Presumably this means the line above can get dropped? Will webpacker try to process the flags twice otherwise?